sdk/schemaregistry/schema-registry-json/review/schema-registry-json.api.md

```diff
@@ -6,10 +6,15 @@
 
 import { SchemaRegistry } from '@azure/schema-registry';
 
+// @public
+export interface DeserializeOptions {
+    validate?: (message: string, schema: string) => void;
```

Comment:
nit: maybe `validationCallback` or something would be more clear?

Reply:
I'll make the update. We're discussing on the design of the validate callback, so I'll update the README and the samples once we finalize. I'm currently working on the test suite in another branch, so I'll add the unit test for it there.